### PR TITLE
January Patch 3!

### PR DESCRIPTION
**Pull request by RadiantFlash7**
Fixes map issues, See commit 
8fa1d50
Enables viruses for a test period to see if it's a practical event to run on server,
Replaces major carp migration with major spider migration.
Fluffifies events for cryogaia, requests security patrol for hostile events.

### CODE REVIEW

File: code/modules/events/radiation_storm.dm [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
-	command_announcement.Announce("High levels of radiation detected near \the [station_name()]. Please evacuate into one of the shielded maintenance tunnels or dorms.", "Anomaly Alert", new_sound = 'sound/AI/radiation.ogg') //VOREStation Edit - Dorms ref
+	command_announcement.Announce("Stellar Radiation flare detected from BD +33 2642, Protoplanetary Nebula. Radiation expected to flood space near \the [station_name()]. Please evacuate into one of the shielded maintenance tunnels or dorms.", "Anomaly Alert", new_sound = 'sound/AI/radiation.ogg') //VOREStation Edit - Dorms ref
 
 /datum/event/radiation_storm/start()
 	make_maint_all_access()
 
 /datum/event/radiation_storm/tick()
 	if(activeFor == enterBelt)
-		command_announcement.Announce("The station has entered the radiation belt. Please remain in a sheltered area until we have passed the radiation belt.", "Anomaly Alert")
+		command_announcement.Announce("The outpost has entered the radiation flare. Please remain in a sheltered area until the radiation flare has past.", "Anomaly Alert")
```
Comment (Shadow-Quill) [inline code/modules/events/radiation_storm.dm]:
```suggestion
		command_announcement.Announce("The outpost has entered the radiation flare. Please remain in a sheltered area until the radiation flare has passed.", "Anomaly Alert")
```

File: code/modules/events/rogue_drones.dm [hunk 2]
```diff
@@ -23,20 +23,21 @@
 	var/msg
 	var/rng = rand(1,5)
 	//VOREStation Edit Start TFF 16/12/19 - Sif -> Virgo 3b
+	//YW EDIT 1/13/2020 - Virgo3b -> Cryogaia
 	switch(rng)
 		if(1)
-			msg = "A combat drone wing operating in close orbit above Virgo 3b has failed to return from a anti-piracy sweep.  If any are sighted, \
+			msg = "A Prototype combat drone wing operating in close orbit above Borealis 1 has failed to return from a anti-piracy sweep.  If any have survived re-entry. \
```
Comment (Shadow-Quill) [inline code/modules/events/rogue_drones.dm]:
```suggestion
			msg = "A prototype combat drone wing operating in close orbit above Borealis 1 has failed to return from a anti-piracy sweep.  If any have survived re-entry. \
```

Comment (RadiantFlash7) [reply code/modules/events/rogue_drones.dm]: I am a derp. Will change that when I get home. Thanks.

File: code/modules/events/rogue_drones.dm [hunk 3]
```diff
@@ -23,20 +23,21 @@
 	var/msg
 	var/rng = rand(1,5)
 	//VOREStation Edit Start TFF 16/12/19 - Sif -> Virgo 3b
```
Comment (TheFurryFeline) [inline code/modules/events/rogue_drones.dm]: Feel free to remove this comment portion since you're making your own edit to the event.

**Comment by Hawk-v3** [comment]
@RadiantFlash7 Please check commented suggestions.

**Comment by RadiantFlash7** [comment]
Yep. Will make da changes when I get home

**Comment by RadiantFlash7** [comment]
Have not yet made changes, Had class yesterday, was not able to complete dev work.

**Comment by RadiantFlash7** [comment]
Please lordy, Don't be break on me now.

**Pull request merged**